index.html

```diff
@@ -6,6 +6,7 @@
     <link rel="stylesheet" type="text/css" media="screen" href="./CSS/index.css" />
     <script src="./Menu/Menu.js" async></script>
     <script src="./Article/Article.js" async></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/2.0.2/TweenMax.min.js"></script>
```

Comment:
this might be the cause of some issues, you might need to apply the async keyword in this as well

Reply:
thanks for pointing this out! should be fixed now